packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -655,9 +677,12 @@ module.exports = function(webpackEnv) {
       // It is absolutely essential that NODE_ENV is set to production
       // during a production build.
       // Otherwise React will be compiled in the very slow development mode.
-      new webpack.DefinePlugin(env.stringified),
+      new webpack.DefinePlugin({
+        ...env.stringified,
+        'typeof window': '"undefined"',
```

Comment:
Could you confirm what this is doing?

Reply:
https://webpack.js.org/plugins/define-plugin/#usage - defining `typeof window` is the closest I've seen to a 'standard' way of stripping out client/server-only code from their respective bundles.